apps/vercel/frontend/src/components/config-screen/ContentTypePreviewPathSection/ContentTypePreviewPathSelectionRow/ContentTypePreviewPathSelectionRow.tsx

```diff
@@ -63,6 +64,9 @@ export const ContentTypePreviewPathSelectionRow = ({
     onRemoveRow(configuredContentTypePreviewPathSelection);
   };
 
+  const validatePreviewPath = (previewPath: string) =>
+    [/^\/.*{([^}]+)}.*$/].some((regex) => regex.test(previewPath));
```

Comment:
require the string to start with a '/' and include at least one set of '{}' within the string. Thoughts on this logic for the validation? Do we want to include or exclude a pattern?

Reply:
Hm, the rules we are validating and the logic make sense but I think going with a single regex means we lack meaningful information about why the particular string fails validation.

I recommend we isolate our rules, test them separately, and collect error messages indepedently. So right now we have 3 rules:

* Path cannot be blank
* Path must start with a `/` (i'm not 100% sure about this rule btw, since we can always join the path to the main domain in our redirect and add the '/' ourselves)
* Path must contain at least one "token" (still think we need a better word for this)

Reply:
I have adjusted the wording of the main validation! The word "token" is a TBD adjustment